tests/Feature/Http/Controllers/Auth/PasswordResetTest.php

```diff
@@ -87,7 +87,7 @@ public function testPasswordCanBeResetWithValidToken(): void
         Notification::fake();
 
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````